src/index.tsx

```diff
@@ -159,6 +163,7 @@ export function NhostApolloProvider({
   children,
   headers = {},
   publicRole = 'public',
+  fetchPolicy = 'cache-and-network',
```

Comment:
We should have the same defaults as the Apollo client -> 'cache-first'

https://www.apollographql.com/docs/react/data/queries/#:~:text=this%20is%20the%20default%20fetch%20policy.

Reply:
True!

Reply:
I used  `undefined` as the default value which will then allow Apollo to set their own default value (`cache-first`).